src/crab/interval.cpp

```diff
@@ -200,6 +200,8 @@ interval_t interval_t::And(const interval_t& x) const {
             return interval_t((*left_op) & (*right_op));
         } else if (lb() >= 0 && x.lb() >= 0) {
```

Comment:
The reasoning here can become confusing if it can be performed between an svalue and a uvalue, so I guess a comment requiring it to be the same may be a good idea (and maybe we should, in the future, separate the two kinds of intervals by type?).

Reply:
Added comment.  This should always operate on uvalues.

Reply:
So when is it not >= 0?

Reply:
I think the only case is when it is TOP

Reply:
So I think it's better to test and assert

Reply:
See updated code.

Reply:
Ok, added asserts